site/package.json

```diff
@@ -93,6 +93,8 @@
     "@types/react-slick": "^0.23.7",
     "@types/uuid": "^8.3.4",
     "babel-plugin-inline-react-svg": "2.0.1",
+    "chalk": "^4.1.2",
+    "execa": "^5.1.1",
```

Comment:
We need to switch to `"type": "module"` in `package.json` to upgrade to chalk 5 and execa 6 (they are ESM-only). We can do this when https://github.com/vercel/next.js/pull/33637 is merged and we upgrade Next.js to latest.